metrics/api/src/main/java/io/helidon/metrics/api/SystemTagsManagerImpl.java

```diff
@@ -180,7 +188,12 @@ public Iterable<Map.Entry<String, String>> withScopeTag(Iterable<Map.Entry<Strin
 
     @Override
     public Iterable<Tag> displayTags() {
-        return List.copyOf(systemTags);
+        return List.copyOf(systemTags.get());
+    }
+
+    @Override
+    public Map<String, String> displayTagPairs() {
+        return Map.copyOf(systemTagPairs);
```

Comment:
A copy? Or would an unmodifiable view suffice?

Reply:
No need for a copy here. Changed.